src/main/java/com/microsoft/sqlserver/jdbc/dtv.java

```diff
@@ -738,11 +738,15 @@ private void sendTemporal(DTV dtv, JavaType javaType, Object value) throws SQLSe
             if (null != typeInfo) // updater
             {
                 switch (typeInfo.getSSType()) {
+                    case DATETIME:
                     case DATETIME2:
                         // Default and max fractional precision is 7 digits (100ns)
+                        // Send DateTime2 to DateTime columns to let the server handle nanosecond rounding. Also
+                        // adjust scale accordingly to avoid rounding on driver's end.
```

Comment:
sorry to nitpick, but multi-line comments should be in the /* */ format

Reply:
done.